external fun foo() {
    break

    while True {
        // Good
        break

        // Bad
        fun() { break }
    }

}

// args: check
// expected exit status: 1
// expected stdout:
// Error: `break` can only be used inside loops.
// ---| src/test_files/check/break_without_loop.gdn:2:5
// 2  |     break
//    |     ^^^^^
// 
// Error: `break` can only be used inside loops.
// ---| src/test_files/check/break_without_loop.gdn:9:17
// 9  |         fun() { break }
//    |                 ^^^^^

